src/main/java/bio/terra/cli/command/helperclasses/options/NotebookName.java

```diff
@@ -0,0 +1,65 @@
+package bio.terra.cli.command.helperclasses.options;
+
+import bio.terra.cli.command.exception.UserActionableException;
+import bio.terra.cli.context.GlobalContext;
+import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.service.WorkspaceManager;
+import bio.terra.cloudres.google.notebooks.InstanceName;
+import bio.terra.workspace.model.GcpAiNotebookInstanceAttributes;
+import bio.terra.workspace.model.ResourceDescription;
+import picocli.CommandLine;
+
+/**
+ * Command helper class for identifying a notebook by either the resource name of the GCP instance
+ * name in `terra notebooks` commands.
+ *
+ * <p>This class is meant to be used as a @CommandLine.Mixin.
+ */
+public class NotebookName {
```

Comment:
nit: Maybe rename this so it's clearer that this isn't just a name option -- `NotebookNameOrId`? `NotebookInstance`? Or something else you think is better -- it just seems misleading because `ResourceName` is a single option so this one sounds like it is too.

Reply:
Yeah, naming's hard. Went with `NotebookInstance`